### DISCUSSION TITLE
Setting avif quality settings for comic files

### DISCUSSION THREAD

**Discussion by zjDrummond** [ImageMagick/ImageMagick#6734] [Help]
### ImageMagick version

7.1.0.19

### Operating system, version and so on

windows 11 22h2, 22621.2283

### Description

I'm using a tool that incorporates imagemagick for its converting of comicbook archive files. Comic archives are zip archives with custom .cbz extensions which contain numbered images for a comic book reader app to view. 

I have been using the app for webp conversion of the jpg files with the quality setting at 75. However this quality setting doesn't seem right for avif. What would be a comparable quality setting for avif? 

Hope this is a good place to put this question. Thanks.

**Comment by fmw42** [comment]
Not sure anyone can answer unless they have experimented.  I think you should just try different settings and find the one that seems best to you.
